external-task-worker/src/main/java/info/novatec/micronaut/camunda/externaltask/worker/TestHandler.java

```diff
@@ -0,0 +1,37 @@
+package info.novatec.micronaut.camunda.externaltask.worker;
+
+import org.camunda.bpm.client.task.ExternalTask;
+import org.camunda.bpm.client.task.ExternalTaskHandler;
+import org.camunda.bpm.client.task.ExternalTaskService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Singleton;
+import java.util.HashMap;
+import java.util.Map;
+
+@Singleton
+@ExternalTaskSubscription(topic = "my-topic", lockDuration = "1000")
+public class TestHandler implements ExternalTaskHandler {
+
+    private static final Logger log = LoggerFactory.getLogger(TestHandler.class);
+
+    @Override
+    public void execute(ExternalTask externalTask, ExternalTaskService externalTaskService) {
```

Comment:
Maybe it would be nicer if "ExternalTaskService" was a bean which we inject. Seems to be a cleaner interface...